man/syncthing-config.5

```diff
@@ -607,6 +607,10 @@ If the address is an absolute path it is interpreted as the path to a UNIX socke
 (Added in v0.14.52.)
 .UNINDENT
 .TP
+.B unixSocketPermissions
```

Comment:
That's autogenerated from the documentation repo, and I think you already have a PR line up for that - i.e. no need to modify the manpage here.

Reply:
Done.